### PR TITLE
PST-576 readonly flag

### PR DESCRIPTION
**Pull request by pivnicek**
since the workspace is created in the construction of the dataloader class there is no way that I can see to inject a mock of the components class in there for testing purposes.  So I'm just checking the log message :-/

### CODE REVIEW

**Comment by pivnicek** [comment]
ok, that's slightly better, but I'm guessing you won't like to much the `getWorkspaceId` 😄

**Comment by pivnicek** [comment]
I believe this should be green now.

**Comment by pivnicek** [comment]
oops, forgot to update the configuration test

File: src/Docker/Runner/DataLoader/DataLoader.php [hunk 1]
```diff
@@ -311,6 +312,24 @@ public function getWorkspaceBackendSize(): ?string
         return null;
     }
 
+    public function getWorkspaceId(): ?string
```
Comment (odinuv) [inline src/Docker/Runner/DataLoader/DataLoader.php:315]: Ouch, That's just for tests

Comment (pivnicek) [reply src/Docker/Runner/DataLoader/DataLoader.php:315]: yeah :-(

**Comment by pivnicek** [comment]
Thanks odin

**Review by odinuv — APPROVED** [review]

**Pull request merged**